app/helpers/status_tag_helper.rb

```diff
@@ -9,6 +9,18 @@ def review_section_tag(job_application, step)
     end
   end
 
+  def vacancy_draft_status(organisation, vacancy, section)
+    step_process = Publishers::Vacancies::VacancyStepProcess.new(:review, organisation: organisation, vacancy: vacancy)
+
+    active_steps = step_process.step_groups[section]
+    completed_steps = vacancy.completed_steps.map(&:to_sym).intersection(active_steps)
+    if active_steps - completed_steps == []
+      { status: :completed, colour: "blue" }
```

Comment:
:raising_hand_man: This would set a blue tag, right?

I think the "**Completed**" tag has no colour anymore as per ticket examples: 
![image](https://github.com/user-attachments/assets/246a9925-24d9-43d8-8b3a-e0eb59861a1b)

The `#review_section_tag` helper was recently updated by @KyleMacPherson to reflect this.